compiler/luci/service/src/Nodes/CircleSoftmax.cpp

```diff
@@ -14,8 +14,12 @@
  * limitations under the License.
  */
 
+#include <luci/Service/CircleShapeInference.h>
+
 #include "CircleCloneNode.h"
 
+#include "CircleShapeInferenceHelper.h"
+
```

Comment:
I've noticed adding includes in  #13780 and found that we need some updates

```suggestion
#include "luci/Service/CircleShapeInference.h"

#include "CircleCloneNode.h"
#include "CircleShapeInferenceHelper.h"

```

I couldn't find how the relation of this source and headers was... and now I see.

order of headers;
- direct header for this source file with `"..."`, should be in same module
- in same folder with `"..."`
- in another another module with `<...>`